models/classes/class.QtiTestConverter.php

```diff
@@ -311,6 +316,11 @@ private function arrayToComponent(array $testArray, QtiComponent $parent = null,
         }
     }
 
+    private function removeScriptTags(string $value): string
+    {
+        return preg_replace('/<*script[^>]*>.*<\/script[^>]*>*/is', '', $value);
```

Comment:
Please make sure that the $value given here has been converted with `htmlentities`. Code wouldn't catch `&lt;script&gt;` here.